app/src/Controller/Webhooks/Mailgun.php

```diff
@@ -40,8 +40,16 @@ public function hook(Request $request, Response $response)
 			$email = \Dappur\Model\Emails::where('secure_id', $messageId)->first();
 
 			if ($email) {
-				$email->status = $payload->{'event-data'}->event;
-				$email->save();
+
+                $addStatus = new \Dappur\Model\EmailsStatus;
+                $addStatus->status = $payload->{'event-data'}->event;
+
+                switch ($payload->{'event-data'}->event) {
+                    case 'clicked':
+                        $addStatus->details = $payload->{'event-data'}->url;
+                        break;
+                }
+				$addStatus->save();
```

Comment:
__[PHP_CodeSniffer]__ <sub>posted by Sider</sub>

Spaces must be used to indent lines; tabs are not allowed